### PR TITLE
GEOPY-1530: Use UIJson group 

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1530 - Use UIJson group **

### CODE REVIEW

File: geoh5py/shared/utils.py [hunk 1]
```diff
@@ -682,3 +695,27 @@ def update(self, value: dict, **kwargs) -> None:  # type: ignore
                 val = self[key].union(val)
             value[key] = val
         super().update(value, **kwargs)
+
+
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/shared/utils.py:699]: It's a good point. We haven't encountered it yet, but we should

[the remark this answers is not in the document]
Comment (MatthieuCMira) [reply geoh5py/shared/utils.py:699]: ahahah, silly me:

np.isfinite(np.nan) ==False.

So nan2str is redundant...

**Review by MatthieuCMira — APPROVED** [review]
oupsi, it was okay on the first run

**Comment by domfournier** [comment]
Don't it is. The inf2str would have returned nan, which would be bad for ui.json. The new `nan2str` returns an empty string instead.

**Comment by MatthieuCMira** [comment]
> Don't think it is. The inf2str would have returned nan, which would be bad for ui.json. The new `nan2str` returns an empty string instead.

Ok, but then nantostr should pass before inftostr in your list

**Comment by domfournier** [comment]
Yep

**Pull request merged**